editor/plugins/script_text_editor.cpp

```diff
@@ -690,7 +690,14 @@ void ScriptTextEditor::_update_bookmark_list() {
 	bookmarks_menu->add_separator();
 
 	for (int i = 0; i < bookmark_list.size(); i++) {
+		// Strip edges to avoid rendering unknown ASCII symbol ('�')
 		String line = code_editor->get_text_edit()->get_line(bookmark_list[i]).strip_edges();
+
+		// Strip edges from comments' content
+		if (line.length() > 0 && line.get(0) == '#') {
+			line = "# " + line.substr(1).strip_edges();
```

Comment:
Won't this crash if `line.length() == 1`?

Reply:
But beyond that, I'm not sure why `strip_edges()` is used to solve this. `strip_edges()` removes whitespace characters at the start and end of the string (and here if it's working I guess invalid chars are mistakenly seen as "whitespace"), but won't handle any invalid characters within the string. So `# Some comment<TAB>More comment` will still have the same problem.

Reply:
I wholeheartedly agree.
In the beginning, I thought about understanding why are escape characters are rendered improperly in the first place but quickly got overwhelmed by the size of the codebase.
When I find something that solves the issue in the more general case, I will add commits to this pull request and let you know

Reply:
`replace("\t", "    ")`

Reply:
I tried to render other escape characters with the same result and I noticed that this behaviour applies to every character that comes before the space in ASCII (hex 0x20, decimal 32).
Is this intended behaviour? Or should we just replace them with whitespace?